src/main/java/pages/NewTabPage.java

```diff
@@ -0,0 +1,26 @@
+package pages;
+
+import map.Locators;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class NewTabPage {
+    private WebDriver driver;
+
+    public NewTabPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    // Verifying if the expected elements are visible in the new tab.
+    public boolean verifyTabButtons() {
+        return driver.findElement(Locators.homelink).isDisplayed() &&
+                driver.findElement(Locators.courseslink).isDisplayed() &&
```

Comment:
Correct the naming of the functions and variables and use camelCase(coursesLink)